### PR TITLE
Update Network Getting Started for FQCN/collection world

### PR DESCRIPTION
**Pull request by samccann**
##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

docs.ansible.com
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: docs/docsite/rst/network/getting_started/network_roles.rst [hunk 1]
```diff
@@ -27,18 +27,18 @@ To demonstrate the concept of what a role is, the example ``playbook.yml`` below
    ---
    - name: configure cisco routers
      hosts: routers
-     connection: network_cli
+     connection: ansble.netcommon.network_cli
```
Comment (bmillemathias) [inline docs/docsite/rst/network/getting_started/network_roles.rst]: typo `ansble`

File: docs/docsite/rst/network/getting_started/network_resources.rst [hunk 2]
```diff
@@ -32,9 +32,8 @@ Ansible hosts module code, examples, demonstrations, and other content on GitHub
 
 - `Network-Automation <https://github.com/network-automation>`_ is an open community for all things network automation. Have an idea, some playbooks, or roles to share? Email ansible-network@redhat.com and we will add you as a contributor to the repository.
 
-- `Ansible <https://github.com/ansible/ansible>`_ is the main codebase, including code for network modules
+- `Ansible collections <https://github.com/ansible-collections>`_ is the main repository for Ansible-maintained and community collectoins, including collections for network devices.
```
Comment (bmillemathias) [inline docs/docsite/rst/network/getting_started/network_resources.rst]: typo `collectoins`

File: docs/docsite/rst/network/getting_started/first_playbook.rst [hunk 3]
```diff
@@ -45,22 +46,22 @@ To confirm your credentials, connect to a network device manually and retrieve i
...
+   ansible all -i vyos.example.net, -c ansible.netcommon.network_cli -u my_vyos_user -k -m vyos.vyos.vyos_facts -e ansible_network_os=vyos.vyos.vyos
 
 The flags in this command set seven values:
   - the host group(s) to which the command should apply (in this case, all)
   - the inventory (-i, the device or devices to target - without the trailing comma -i points to an inventory file)
   - the connection method (-c, the method for connecting and executing ansible)
   - the user (-u, the username for the SSH connection)
   - the SSH connection method (-k, please prompt for the password)
-  - the module (-m, the ansible module to run)
+  - the module (-m, the ansible module to run, using the fully qualified collection name (FQCNO))
```
Comment (bmillemathias) [inline docs/docsite/rst/network/getting_started/first_playbook.rst]: typo: `FQCNO` instead of `FQCN`

**Review by Qalthos — APPROVED** [review]
All seems good to me

**Pull request merged**